README.md

```diff
@@ -0,0 +1,81 @@
+# Sumo Logic OpenTelemetry auto-instrumentation for JavaScript
+
+The [Sumo Logic](https://www.sumologic.com/) OpenTelemetry auto-instrumentation for JavaScript library enables tracing in the browser.
+
+## Installation
+
+The easiest way to start collecting traces from yor website is to put the code below inside the `<head></head>` tags on your website:
```

Comment:
It sounds unlikely to me package users would want to add a large blocking script to the head of their site. Can we propose a more realistic solution?